src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1053,6 +1076,15 @@ export default {
 				size: this.isCompact ? AVATAR.SIZE.COMPACT : AVATAR.SIZE.DEFAULT,
 			}
 		},
+
+		getFilterLabel(filter) {
+			if (filter === 'unread') {
+				return t('spreed', 'Unread')
+			} else if (filter === 'mentions') {
+				return t('spreed', 'Mentions')
+			}
+			return ''
+		},
```

Comment:
```suggestion
const FILTER_LABELS = {
	unread: t('spreed', 'Unread'),
	mentions: t('spreed', 'Mentions'),
	defult: '',
} as const
```

and then `FILTER_LABELS[filter]`